token.sol

```diff
@@ -278,7 +278,7 @@ contract token is safeMath, module, announcementTypes {
      * @param _extraData Data to give forward to the receiver
      * @return Whether the transfer was successful or not
      */
-    function transferAndCall(address _to, uint256 _amount, bytes _extraData) isReady external returns (bool) {
+    function transfer(address _to, uint256 _amount, bytes _extraData) isReady external returns (bool) {
```

Comment:
Similar to `transfer` function that I have recently described. You shouldn't just rename `transferAndCall` into `transfer` function.It will not work.